code/game/objects/items/weapons/storage/backpack.dm

```diff
@@ -370,6 +370,11 @@
 	var/zipped = TRUE
 	/// How long it takes to toggle the zip state of this bag
 	var/zip_time = 0.7 SECONDS
+	/// If it has a sprite when open, change the icon state to it
+	var/open_icon_sprite
+	var/open_item_sprite
+	/// Do we want the bag to be antidropped when zipped up?
```

Comment:
Your comment is more what we'd expect from a proc, not a var. Please comment both vars with appropriate descriptors